src/PlaywrightSharp.Tests/Frame/FrameManagementTests.cs

```diff
@@ -213,19 +216,19 @@ public async Task ShouldReportDifferentFrameInstanceWhenFrameReAttaches()
                 window.frame.remove();
             }");
             Assert.True(frame1.Detached);
-            var frameattached = new TaskCompletionSource<IFrame>();
+            var frameAttached = new TaskCompletionSource<IFrame>();
             void WaitFrameAttached(object sender, FrameEventArgs e)
             {
-                frameattached.TrySetResult(e.Frame);
+                frameAttached.TrySetResult(e.Frame);
                 Page.FrameAttached -= WaitFrameAttached;
             }
             Page.FrameAttached += WaitFrameAttached;
 
             await Task.WhenAll(

```

Comment:
use `TaskUtils`

Reply:
As I mentioned in your PR, I don't want to use TaskUtils in the library.

Reply:
these are tests ;)

Reply:
HA!  sooooo. yes 😂